velox/exec/tests/WindowTest.cpp

```diff
@@ -79,6 +80,74 @@ TEST_F(WindowTest, spill) {
   ASSERT_GT(stats.spilledPartitions, 0);
 }
 
+TEST_F(WindowTest, rankWithEqualValue) {
+  auto data = makeRowVector(
+      {"c1"},
+      {makeFlatVector<int64_t>(std::vector<int64_t>{1, 1, 1, 1, 1, 2, 2})});
+
+  createDuckDbTable({data});
+
+  const std::vector<std::string> kClauses = {
+      "rank() over (order by c1 rows unbounded preceding)"};
+
+  auto plan = PlanBuilder()
+                  .values({data})
+                  .orderBy({"c1"}, false)
+                  .streamingWindow(kClauses)
+                  .planNode();
+
+  SCOPED_TESTVALUE_SET(
```

Comment:
Maybe also check that this value wouldn't be set for a non-row streaming function.

Reply:
@aditi-pandit Do you mean to add this check in https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/WindowTest.cpp#L40 https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/WindowTest.cpp#L82 https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/WindowTest.cpp#L142  functions?

Reply:
@JkSelf : Specifically, I was thinking about testing the logic in supportsRowStreaming() function in Window.cpp.

So basically add tests with first_value/nth_value functions and ensure it is not using rowStreaming build.

Reply:
@aditi-pandit Updated. Can you help to review again? Thanks.